src/Types/ModelProperty/GenericModelPropertyType.php

```diff
@@ -65,7 +65,7 @@ public function acceptsWithReason(Type $type, bool $strictTypes): AcceptsResult
             }
 
             if (count($genericType->getObjectClassNames()) < 1) {
-                return AcceptsResult::createNo();
+                return AcceptsResult::createYes();
```

Comment:
Can we do a `maybe` here? Are the any negative impacts?

Reply:
`maybe` results in the same error